registrations/models.py

```diff
@@ -660,6 +660,18 @@ def send_event_cancellation_notifications(self, is_sub_event_cancellation=False)
                 is_sub_event_cancellation=is_sub_event_cancellation,
             )
 
+    def get_calendar_events(self):
+        if self.event.is_recurring_super_event:
+            events = self.event.sub_events.filter(
+                deleted=False,
+                event_status=Event.Status.SCHEDULED,
```

Comment:
I think that both Event.Status.SCHEDULED and Event.Status.RESCHEDULED should be included.

Reply:
Added `RESCHEDULED` status as well.